src/components/scores/export.cjsx

```diff
@@ -19,7 +19,7 @@ ScoresExport = React.createClass
 
   getInitialState: ->
     downloadUrl: null
-    forceDownloadUrl: null
+    finalDownloadUrl: null
     lastExported: null
     tryToDownload: false
     downloadedSinceLoad: false
```

Comment:
I *do* like the added documentation, and you never forget to initialize the state (If you don't define `getInitialState` things will blow up when you do `{downloadUrl} = @state`)